spec/esnext-spec.coffee

```diff
@@ -0,0 +1,27 @@
+{create6to5VersionAndOptionsDigest} = require '../src/esnext'
+crypto = require 'crypto'
+
+describe "esnext", ->
+
+  it "::create6to5VersionAndOptionsDigest", ->
```

Comment:
Usually we put the method name in a `describe` and then the `it` would be something like:

```coffee
it "returns a digest for the library version and specified options", ->
```